src/js/plugins/plugin.mobile_nav.js

```diff
@@ -0,0 +1,220 @@
+            <button class="BRmobileHamburger"></button>
+            <span class="BRtoolbarMobileTitle" title="${escapedTitle}">${this.bookTitle}</span>
+        </span>
+      `;
+      $el
+        .addClass('responsive')
+        .prepend($(toolbar));
+    }
+    return $el;
+  };
+})(BookReader.prototype.buildToolbarElement);
+
+/**
+ * This method builds the html for the mobile drawer. It can be decorated to
+ * extend the default drawer.
+ * @return {jqueryElement}
+ */
+BookReader.prototype.buildMobileDrawerElement = function() {
+  let experimentalHtml = '';
+  if (this.enableExperimentalControls) {
+    experimentalHtml = `
+        <p class="DrawerSettingsTitle">Experimental (may not work)</p>
+        <button class="BRaction default high-contrast-button">Toggle high contrast</button>
+    `;
+  }
+
+  const settingsSection = `
+    <span>
+        <span class="DrawerIconWrapper">
+            <img class="DrawerIcon" src="${`${this.imagesBaseURL}icon_gear.svg`}" alt="settings-icon"/>
+        </span>
+        Settings
+    </span>
+    <div class=DrawerSettingsWrapper>
+        <div class="DrawerSettingsLayoutWrapper">
+            <button class="DrawerLayoutButton one_page_mode">
+                <img src="${`${this.imagesBaseURL}icon_one_page.svg`}" alt="Single Page"/>
+                <br>
+                One Page
+            </button>
+            <button class="DrawerLayoutButton two_page_mode TwoPagesButton">
+                <img src="${`${this.imagesBaseURL}icon_two_pages.svg`}" alt="Two Pages"/>
+                <br>
+                Two Pages
+            </button>
+            <button class="DrawerLayoutButton thumbnail_mode">
+                <img src="${`${this.imagesBaseURL}icon_thumbnails.svg`}" alt="Thumbnails"/>
+                <br>
+                Thumbnails
+            </button>
+        </div>
+        <br>
+        <div class="DrawerSettingsTitle">Zoom</div>
+        <button class='BRicon zoom_out'></button>
+        <button class='BRicon zoom_in'></button>
+        <br style="clear:both"><br><br>
+        ${experimentalHtml}
+    </div>
+  `;
+  const moreInfo = `
+    <span>
+        <span class="DrawerIconWrapper ">
+            <img class="DrawerIcon" src="${`${this.imagesBaseURL}icon_info.svg`}" alt="info-icon"/>
+        </span>
+        About This Book
+    </span>
+    <div class="BRmobileInfo"></div>
+  `;
+  const share = `
+    <span>
+        <span class="DrawerIconWrapper">
+            <img class="DrawerIcon" src="${`${this.imagesBaseURL}icon_share.svg`}" alt="info-share"/>
+        </span>
+        Share This Book
+    </span>
+    <div class="BRmobileShare"></div>
+  `;
+  const navMenu = `
+    <nav id="BRmobileMenu" class="BRmobileMenu">
+        <ul>
+            <li class="br-mmenu settings">${settingsSection}</li>
+            <li class="br-mmenu more-info BRmobileMenu__moreInfoRow">${moreInfo}</li>
+            <li class="br-mmenu share">${share}</li>
```

Comment:
These names are too general; they're likely to conflict with something else (especially when in archive.org). Let's try this

```suggestion
            <li class="BRmobileMenu__settings">${settingsSection}</li>
            <li class="BRmobileMenu__moreInfoRow">${moreInfo}</li>
            <li class="BRmobileMenu__share">${share}</li>
```

Reply:
ok. will update locally as they connect with unit test